base/repl/REPL.jl

```diff
@@ -266,20 +267,19 @@ specialdisplay(r::LineEditREPL) = r.specialdisplay
 specialdisplay(r::AbstractREPL) = nothing
 terminal(r::LineEditREPL) = r.t
 
-LineEditREPL(t::TextTerminal, envcolors = false) =  LineEditREPL(t,
-                                              true,
-                                              Base.text_colors[:green],
-                                              Base.input_color(),
-                                              Base.answer_color(),
-                                              Base.text_colors[:red],
-                                              Base.text_colors[:yellow],
-                                              false, false, false, envcolors)
-
-mutable struct REPLCompletionProvider <: CompletionProvider; end
-
-mutable struct ShellCompletionProvider <: CompletionProvider; end
+LineEditREPL(t::TextTerminal, envcolors::Bool=false) =
+    LineEditREPL(t, true,
+        Base.text_colors[:green],
+        Base.input_color(),
+        Base.answer_color(),
+        Base.text_colors[:red],
+        Base.text_colors[:yellow],
+        false, false, false, envcolors
+    )
 
-struct LatexCompletions <: CompletionProvider; end
+mutable struct REPLCompletionProvider <: CompletionProvider end
+mutable struct ShellCompletionProvider <: CompletionProvider end
```

Comment:
Need these two `struct`s continue to be mutable? Above (around old lines 80-84) you changed a couple similar `mutable struct`s to `struct`s, so thought I might ask :).